myia/ir/print.py

```diff
@@ -19,8 +19,12 @@ def _print_node(node, buf, nodecache, offset=0):
     assert node.is_apply()
     print(f"{o}{nodecache(node)} = ", end="", file=buf)
     print(f"{nodecache(node.fn)}(", end="", file=buf)
+    args, kwargs = node.args()
+    args = [nodecache(a) for a in args]
+    args.extend(f"{k}={nodecache(v)}" for k, v in kwargs.items())
+
     print(
-        ", ".join(nodecache(a) for a in node.inputs),
+        ", ".join(args),
         end="",
         file=buf,
```

Comment:
```suggestion
        file=buf
```

I believe that'll make `black` collapse the call into a single line.